classwork_1.js

```diff
@@ -0,0 +1,32 @@
+// 1
+const cutString = (str, n) => {
+    if (str.length > n) {
+       return `${str.slice(0, n)}...` 
+    }
+    else return str
+    }
+    console.log(cutString('Hello', 3))
+    
+    // 2
+    const findRepeatedElem = (arr) => {
+    
+    const obj = arr.reduce((prev, cur) => {
+            prev[cur] = (prev[cur] || 0) + 1
+            return prev
+        }
+    ,{})
+    const keys = Object.keys(obj)
+    const values = Object.values(obj)
+    const maxValue = Math.max(...values)
+    const maxIndex = values.findIndex(item => item === maxValue)
+    return keys[maxIndex]
```

Comment:
your function should return number not a string :) Just convert your result into number